static/app/views/organizationIntegrations/sentryAppExternalForm.tsx

```diff
@@ -295,11 +297,6 @@ export class SentryAppExternalForm extends Component<Props, State> {
         filterOption,
         allowClear,
       };
-      // default message for async select fields
-      if (isAsync) {
-        fieldToPass.noOptionsMessage = () => 'Type to search';
```

Comment:
Does this text still exist somewhere else?

Reply:
I figured it was fine to remove it since even when you search gibberish, showing a message saying "type to search" is less helpful than "no options".  You're right though, it should still be around to guide the user when they first see the field, so I set it as the placeholder